Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -385,7 +387,9 @@
 				2E92B989035E25169DF9300616F23AAB /* Environment.swift */,
 				0C6D246B203676EC17FA9473D97CB418 /* Filters.swift */,
 				02B0A9A1128124B1A17D5AB1E23A0A45 /* Filters+Numbers.swift */,
-				47DB41142F62E8B5174F3DF281DCD5E3 /* Filters+Strings.swift */,
+				6D1EF02F1F49EC7C00EA91EF /* Filters+Strings+Lettercase.swift */,
+				6D1EF02B1F49E85D00EA91EF /* Filters+Strings+Mutations.swift */,
```

Comment:
It happens when you add the files manually to the pods project. When you run `pod install` you get the longer UUIDs.